grpc-gateway/pb/retrieveResourceFromDevice.go

```diff
@@ -0,0 +1,41 @@
+package pb
+
+import (
+	"context"
+
+	"github.com/gofrs/uuid"
+	"github.com/plgd-dev/cloud/resource-aggregate/commands"
+	"github.com/plgd-dev/cloud/resource-aggregate/events"
+	"google.golang.org/grpc/peer"
+)
+
+func (req *RetrieveResourceFromDeviceRequest) ToRACommand(ctx context.Context) (*commands.RetrieveResourceRequest, error) {
```

Comment:
I don't like much that we are mapping commands to different gRPC contracts. We can reuse existing commands and in case we will need to change them and we will not want to introduce a breaking change, then we can do mapping. But I think now it only creates unnecessary boilerplate.

Reply:
I this function I set correltaionID and ConnectionID to RetrieveResourceRequest so it is not same.